### PR TITLE
Fix Windows PATH setting for Python configuration

### LINKED ISSUE #474: Fails to find configured Python on Windows
This is a
 - [x] Bug Report

<br/>
If this is a bug report, What are the steps to reproduce?

If I configure the `python` option, nexe still fails to find the configured Python.  It turns out this is an issue with the way the `process.env` object is being replicated in "compiler.ts" because, while part of the `process.env` object, the properties are treated caseless-ly (like when using the `caseless` module) but once they are copied over to `this.env`, they are no longer caseless. In my case, `this.env.PATH` is `undefined` because the property name after assignment was actually `this.env.Path`.

```
// relevant snippet only:
{
   "python": "C:/python27"
}
```

<br/><br/>
Please also provide:

- Platform(OS/Version): Windows 10 Enterprise x64
- Host Node Version: `8.11.1`
- Target Node Version: `4.8.3`
- Nexe version: `2.0.0-next.9`
- Python Version: `2.7.11`

### PR DESCRIPTION
**Pull request by JamesMGreene**
Fixes #474

### CODE REVIEW

File: src/compiler.ts [hunk 1]
```diff
@@ -58,8 +58,13 @@ export class NexeCompiler {
     this.log.step('nexe ' + version, 'info')
     if (python) {
       if (isWindows) {
-        this.env.PATH = '"' + dequote(normalize(python)) + '";' + this.env.PATH
```
Comment (JamesMGreene) [inline src/compiler.ts]: @calebboyd et al: if you don't like the "shuffling" here, we can make use of the [`caseless` module](https://www.npmjs.com/package/caseless) instead.

**Comment by JamesMGreene** [comment]
I'll rename `winPath` to `originalPath` as @calebboyd advised on gitter, and then resubmit this PR (hopefully tonight) with the changes made against the `dev` branch instead of `no-bundle`.

If you have any other feedback on the actual changes here, please feel free to let me know still. :+1:

File: src/compiler.ts [hunk 2]
```diff
@@ -58,8 +58,13 @@ export class NexeCompiler {
...
+        // Do a little shuffling to correctly set the PATH regardless of property name case sensitivity
+        let winPath = process.env.PATH
```
Comment (calebboyd) [inline src/compiler.ts]: `const originalPath`

**Comment by JamesMGreene** [comment]
Closing. Superseded by #476.

**Pull request closed without merging**